src/Assets/TestWorkloads/manifests/Microsoft.NET.Sdk.TestWorkload/WorkloadManifest.json

```diff
@@ -16,6 +16,19 @@
   },
   "packs": {
     "Microsoft.NET.Sdk.TestWorkload.Pack": {
+      "kind": "sdk",
+      "version": "0.42.42",
+      "alias-to":
+      {
+        "win": "Microsoft.NET.Sdk.TestWorkload.Pack.Win",
+        "unix": "Microsoft.NET.Sdk.TestWorkload.Pack.Unix"
+      }
+    },
+    "Microsoft.NET.Sdk.TestWorkload.Pack.Win": {
```

Comment:
packs that are aliased _to_ do not need to be included in the manifest, unless they also need to be referenced directly by workloads